ab-testing/abTest.ts

```diff
@@ -19,4 +19,8 @@ import type { ABTest } from './types';
  * - 100% Test	variant		MVT 500-999
  */
 
-export const ABTests: ABTest[] = [];
+const ABTests: ABTest[] = [];
+
+const activeABtests = ABTests.filter((test) => test.status === 'ON');
```

Comment:
Non-blocking low-priority possibly-daft question for my own curiosity: Should we also filter out tests that have expired at this point?